js/copy-code.js

```diff
@@ -0,0 +1,88 @@
+const COPY_CODE_CLASS = "usa-copy-code";
+const COPY_BUTTON_CLASS = `usa-button usa-button--outline ${COPY_CODE_CLASS}__button`;
+const COPY_BUTTON_SUCCESS_CLASS = `${COPY_CODE_CLASS}__button--success`;
+const COPY_WRAPPER_CLASS = `${COPY_CODE_CLASS}__wrapper`;
+
+const COPY_CODE = document.querySelectorAll(`.${COPY_CODE_CLASS}`);
+
+/**
+ * Creates a wrapper for copy button.
+ *
+ * @return {HTMLElement} - A div for `usa-copy-code__wrapper`.
+ */
+const createWrapper = () => {
+  const wrapper = document.createElement("div");
+  wrapper.classList.add(COPY_WRAPPER_CLASS);
+
+  return wrapper;
+};
+
+/**
+ * Creates a copy button component.
+ *
+ * @return {HTMLElement} - A button for copying code `usa-copy-code__button`.
+ */
+const createCopyButton = () => {
+  const btn = document.createElement("button");
+  const btnText = `
+    <span class="" aria-hidden="true">Copy</span>
+    <span class="usa-sr-only">Copy component code</span>
+  `;
+
+  btn.className = COPY_BUTTON_CLASS;
+  btn.setAttribute("type", "button");
+
+  btn.insertAdjacentHTML("beforeend", btnText);
+
+  return btn;
+};
+
+/**
+ * Copy <code> text content when copy code button is clicked.
+ *
+ * @param {Event} event - The click event from copy code button.
+ * @return {Clipboard} - Code element contents written to clipboard.
+ */
+const copyOnClick = (event) => {
+  const copyBtn = event.currentTarget;
+  const labelVisual = copyBtn.querySelector("[aria-hidden]");
+  const labelSROnly = copyBtn.querySelector(".usa-sr-only");
+
+  // Set success state
+  copyBtn.classList.add(COPY_BUTTON_SUCCESS_CLASS);
+  labelVisual.textContent = "Copied!";
+  labelSROnly.textContent = "Code copied to clipboard";
```

Comment:
Possible small tweak: lead with the verb. Additionally, we might be able to say just the word "Copied" here. I feel like the rest of it is implied. 

```suggestion
  labelSROnly.textContent = "Copied code to clipboard";
```

Reply:
Good points! That makes sense.